dolphin/api/v1/storages.py

```diff
@@ -53,6 +53,15 @@ def validate_parameters(data, required_parameters,
             raise exc_response(explanation=msg)
 
 
+def update_access_info(access_info, body):
+    for attr, value in access_info.__dict__.items():
+        if attr in body:
+            if attr == 'extra_attributes':
+                access_info.extra_attributes.update(body['extra_attributes'])
```

Comment:
Seems extra_attributes will be update twice here

Reply:
it was not .. but now I changed the logic to update the dictionary..